packages/admin/cypress/support/commands.js

```diff
@@ -35,3 +35,15 @@ Cypress.Commands.add('setTinyMceContent', (content, fieldName) => {
   });
   cy.wait(2000);
 });
+
+Cypress.Commands.add('clickButton', (buttonClass, buttonText) => {
+  cy.get(`.${buttonClass}`).should('be.visible').contains(buttonText).click();
+});
```

Comment:
Had it like that initially but figured it may be more flexible if it was generic.  That said, i'm not sure what other buttons we would be clicking without that class. Pros and cons to both i suppose